.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master, release/*, chore/* ]
```

Comment:
Not sure our the objective here, but if its to run on push to PR branch it should have `feature/*` and `push/*`. I would say this is unnecessary though. I think we can remove on push altogether and use branch protection instead.

Reply:
updated